.cicd/eosio-contracts-build.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+set -eo pipefail
+. ./.cicd/helpers/general.sh
+CDT_DIR_HOST=$(pwd)
+mkdir -p build_eosio_contracts 
+
+
+[[ ! -z "$CONTRACTS_VERSION" ]] || export CONTRACTS_VERSION="$(cat "$PIPELINE_CONFIG" | jq -r '.dependencies["eosio.contracts"]')"
+git clone -b "$CONTRACTS_VERSION" https://github.com/EOSIO/eosio.contracts.git 
+
+if [[ $(uname) == 'Darwin' ]]; then
+    export PATH=$CDT_DIR_HOST/build/bin:$PATH
+    cd build_eosio_contracts
+    cmake $CDT_DIR_HOST/eosio.contracts
+    make -j$JOBS
```

Comment:
Also, we should invoke `cmake` using a relative path.
```bash
CMAKE="cmake ../eosio.contracts"
echo "$ $CMAKE"
eval $CMAKE
```
We use a relative path when invoking `cmake` in all of our other build steps. I think it makes it easier to follow along with what the CI system is doing. It also allows developers to copy and paste commands into their terminal, since the absolute paths will not match our workstations.

Reply:
Done.